### PR TITLE
Update stdinbot.py

### PR DESCRIPTION
**Pull request by piecubed**
Add a one second delay after every message to prevent spam.

### CODE REVIEW

File: stdinbot.py [hunk 1]
```diff
@@ -40,5 +40,6 @@ def handle_stdin(irc, hostmask, args):
         if line == '\x04':
             return irc.disconnect(qmsg)
         irc.msg(channels[0], line)
+        time.sleep(1)
```
Comment (luk3yx) [inline stdinbot.py:43]: This delay should be configurable.

**Review by luk3yx — CHANGES_REQUESTED** [review]

File: stdinbot.py (hunk 1 above)
Comment (piecubed) [reply stdinbot.py:43]: True

**Comment by luk3yx** [comment]
That commit was merging my new changes in `master` to your `patch-1`, not merging `patch-1` into master.

**Comment by luk3yx** [comment]
Oh

**Pull request closed without merging**